coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -115,3 +140,23 @@ def get_ignore_comment(self, origin, language):
                         language=language))
                 ignore_comment = None
         return ignore_comment
+
+    def read_ignore_comments(self, original_line, comment_delimiter):
+        """
+        Reads Ignore Comment(s) already present in the line and
```

Comment:
docstrings should use imperative tone per PEP257; i.e. no plural `Reads` & `returns`.

End the imperative command with a full stop.